src/server/pkg/worker/api_server.go

```diff
@@ -497,6 +506,11 @@ func HashDatum(pipelineInfo *pps.PipelineInfo, data []*Input) (string, error) {
 
 // Process processes a datum.
 func (a *APIServer) Process(ctx context.Context, req *ProcessRequest) (resp *ProcessResponse, retErr error) {
+	logger := a.getTaggedLogger(req)
+	logger.Logf("service: worker, method: Process, request: %v, response: %v, err %v, duration: %v", req, nil, nil, 0)
```

Comment:
I'd like to keep the 'started' and 'finished' log lines. I find this helpful for debugging. And for instance here we'd see any calls made when the worker was busy.

If you prefer, we can move these statements below the `worker busy` message where the `Received request` log line is and replace that log line w this pair.